nestkernel/connection_manager.cpp

```diff
@@ -1288,33 +1315,36 @@ nest::ConnectionManager::get_sources( std::vector< index > targets,
 void
 nest::ConnectionManager::get_targets( std::vector< index > sources,
   std::vector< std::vector< index > >& targets,
-  index synapse_model )
+  index synapse_model,
+  std::string post_synaptic_element )
 {
-  thread thread_id;
+  unsigned int thread_id;
   std::vector< index >::iterator source_it;
   std::vector< std::vector< index > >::iterator target_it;
   targets.resize( sources.size() );
   for ( std::vector< std::vector< index > >::iterator i = targets.begin();
         i != targets.end();
-        i++ )
+        ++i )
   {
     ( *i ).clear();
   }
 
-  for ( tVSConnector::iterator it = connections_.begin();
+  /*for ( tVSConnector::iterator it = connections_.begin();
         it != connections_.end();
-        ++it )
+        ++it )*/
```

Comment:
Cosmetic change - remove the commented code. It doesn't contribute to the readability much here. If you think a comment describing the logic here is needed, maybe you can add a descriptive comment instead?